PWGHF/TableProducer/candidateSelectorOmegac0ToOmegaPi.cxx

```diff
@@ -192,9 +214,18 @@ struct HfCandidateSelectorToOmegaPi {
     registry.add("hSelMassCharmBaryon", "hSelMassCharmBaryon;status;entries", {HistType::kTH1F, {axisSel}});
     registry.add("hSelDcaXYToPvV0Daughters", "hSelDcaXYToPvV0Daughters;status;entries", {HistType::kTH1F, {axisSel}});
     registry.add("hSelDcaXYToPvKaFromCasc", "hSelDcaXYToPvKaFromCasc;status;entries", {HistType::kTH1F, {axisSel}});
+    registry.add("hSelCompetingCasc", "hSelCompetingCasc;status;entries", {HistType::kTH1F, {axisSel}});
+    registry.add("hSelKFstatus", "hSelKFstatus;status;entries", {HistType::kTH1F, {axisSel}});
+    registry.add("hSelV0_Casc_Omegacldl", "hSelV0_Casc_Omegacldl;status;entries", {HistType::kTH1F, {axisSel}});
+    registry.add("hSelctauOmegac", "hSelctauOmegac;status;entries", {HistType::kTH1F, {axisSel}});
+    registry.add("hSelChi2GeooverNDFV0_Casc_Omegac", "hSelChi2GeooverNDFV0_Casc_Omegac;status;entries", {HistType::kTH1F, {axisSel}});
+    registry.add("hSelChi2TopooverNDFV0_Casc_Omegac", "hSelChi2TopooverNDFV0_Casc_Omegac;status;entries", {HistType::kTH1F, {axisSel}});
+    registry.add("hSeldecayLenXYOmegac_Casc_V0", "hSeldecayLenXYOmegac_Casc_V0;status;entries", {HistType::kTH1F, {axisSel}});
+    registry.add("hSelcosPaCascToOmegac_V0ToCasc", "hSelcosPaCascToOmegac_V0ToCasc;status;entries", {HistType::kTH1F, {axisSel}});
+    registry.add("hInvMassXiMinus_rej_cut", "hInvMassXiMinus_rej_cut", kTH1D, {{1000, 1.25f, 1.65f}});
   }
 
-  void process(aod::HfCandToOmegaPi const& candidates,
+  void process(soa::Join<aod::HfCandToOmegaPi, aod::HfOmegacKf> const& candidates,
```

Comment:
doing in this way implies that one must have the `HfOmegaKf` table always filled, which is not the case if one uses the DCA fitter. Please templetize the `process` to properly handle this